src/System.Text.Json/tests/JsonReaderTests.cs

```diff
@@ -293,15 +289,15 @@ private static void ReadJsonHelper(JsonReader jsonReader, ICollection<Members> j
 
         private static Value GetValue(ref JsonReader jsonReader)
         {
-            var value = new Value {Type = (Value.ValueType) jsonReader.GetValueType()};
-            var obj = jsonReader.ReadValue();
+            var value = new Value {Type = (Value.ValueType) jsonReader.GetJsonValueType()};
+            var obj = jsonReader.GetValue();
```

Comment:
This is not part of the performance test. JsonReaderTests checks for correctness and hence I need to deserialize to the Json Object and use Json.ToString() to compare input json to output json.